server/server.go

```diff
@@ -194,6 +200,11 @@ func (s *Server) SetSealer(sealer Sealer) {
 	s.sealer = sealer
 }
 
+// RegisterRPC publishes in the server the methods on rcvr.
+func (s *Server) RegisterRPC(rcvr interface{}) error {
+	return s.dispatcher.Register(rcvr)
+}
```

Comment:
Can you explain how this method is used? I'm not sure that I understand it. Who calls this method, and how are the registered methods called?